src/components/Charts/ModelChart.js

```diff
@@ -14,37 +13,23 @@ import Outcomes from '../Outcomes/Outcomes';
 import { COLORS } from 'enums';
 import { formatDate } from 'utils';
 
-const formatIntervention = (intervention, optCase) =>
-  `3 months of ${intervention.toLowerCase()}${optCase || ''}`;
-
-const condensedFormatIntervention = (intervention, optCase) =>
-  `${intervention}${optCase || ''}`;
-
 const isFuture = d => d.x > new Date().valueOf();
 
 const ModelChart = ({
   height,
   condensed,
   projections,
-  forCompareModels, // true when used by CompareInterventions.js component.
+  forCompareModels, // true when used by CompareModels.js component.
 }) => {
-  // We use the inferred projection if supported, otherwise the worst case for the currently active intervention
-  let projection = projections.primary;
-  const currentIntervention = projections.stateIntervention;
-
-  const data = [
-    projections.baseline.getDataset('hospitalizations'),
-    projections.distancingPoorEnforcement.now.getDataset('hospitalizations'),
-    projections.primary.getDataset('hospitalizations'),
-    projections.distancing.now.getDataset('hospitalizations'),
-    projections.baseline.getDataset('beds', 'Available hospital beds'),
-  ];
+  const noActionDataset = projections.baseline.getDataset('hospitalizations');
+  const projectedDataset = projections.projected.getDataset('hospitalizations');
+  const bedsDataset = projections.primary.getDataset('beds');
```

Comment:
The data[] array seemed like unnecessary indirection so I just created direct variables for the datasets we use.